### PR TITLE
Feature/add dcs non vs cmdus

### LINKED ISSUE #1377: [TASK] add EasyMesh R2 CMDUs and TLVs to support DCS
## Description

Task #1152 gathers all the required changes to support DCS aligned to the EasyMesh R2 specifications.
This task is a sub-task that covers the addition of required non-VS CMDUs and TLVs according to the 
R2 specifications.

## Exit criteria

- [x] add the following CMDUs:
CHANNEL_SCAN_REQUEST
CHANNEL_SCAN_RESULT

- [x] add the following TLVs:
tlvChannelScanRequest
tlvChannelScanReport
tlvTimestamp

### PR DESCRIPTION
**Pull request by adam1985d**
This PR introduces the CMDUs and TLVs required to support DCS according to the EasyMesh R2 specifications:

- Added CHANNEL_SCAN_REQUEST_MESSAGE and CHANNEL_SCAN_REPORT_MESSAGE CMDUs as defined in the EasyMesh R2 specifications.
The CMDUs are required to align the DCS feature to Easymesh R2.

- added the 'tlvChannelScanRequest.yaml' file to the source control.
The file implements the TLV class of tlvChannelScanRequest which is added to the CHANNEL_SCAN_REQUEST_MESSAGE CMDU and is defined in the EasyMesh R2 specifications (17.2.39 Table 49)

- added the tlvChannelScanResult.yaml tlvTimestamp.yaml files to the source control.
The files implement the TLV classes of tlvChannelScanResult and tlvTimestamp respectively which are added to the CHANNEL_SCAN_REPORT_MESSAGE CMDU and are defined in the EasyMesh R2 specifications (17.2.40 Table 50 and 17.2.41 Table 51 respectively)

- added the 'tlvChannelScanReportingPolicy.yaml' file to the source control.
The file implements the TLV class of tlvChannelScanReportingPolicy which is added to the MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE CMDU and is defined in the EasyMesh R2 specifications (17.2.37 Table 47)

- added the 'tlvChannelScanCapabilities.yaml' file to the source control.
The file implements the TLV class of tlvChannelScanCapabilities which is added to the AP_CAPABILITY_REPORT_MESSAGE CMDU and is defined in the EasyMesh R2 specifications (17.2.38 Table 48)

Fixes #1377

### CODE REVIEW

File: framework/tlvf/tlvf_conf.yaml [hunk 1]
```diff
@@ -41,6 +41,11 @@ include_yaml_path: {
   "tlvf/wfa_map/tlvSteeringRequest.yaml",
   "tlvf/wfa_map/tlvSupportedService.yaml",
   "tlvf/wfa_map/tlvTransmitPowerLimit.yaml",
+  "tlvf/wfa_map/tlvChannelScanReportingPolicy.yaml",
```
Comment (RanRegev) [inline framework/tlvf/tlvf_conf.yaml]: The entire list is sorted alphabetically, please keep it sorted.

**Review by RanRegev — APPROVED** [review]

File: framework/tlvf/tlvf_conf.yaml (hunk 1 above)
Comment (kantera800) [reply framework/tlvf/tlvf_conf.yaml]: the real question here is why did someone take the effort to sort it :)

**Review by kantera800 — APPROVED** [review]

**Review by morantr — APPROVED** [review]

File: framework/tlvf/tlvf_conf.yaml (hunk 1 above)
Comment (adam1985d) [reply framework/tlvf/tlvf_conf.yaml]: @RanRegev changed to be alphabetically sorted

Comment (RanRegev) [reply framework/tlvf/tlvf_conf.yaml]: > take the effort 

When you work with extreme, powerful, mega editor, this is simple as
`:<from,to> sort u`
With less powerful editors you need to [install extension](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines#:~:text=Usage,has%20the%20default%20hotkey%20F9.). 

Anyhow the benefit across a project is huge: much easier to read and find what you need.
Also, amazingly it helps finding `#include` dependencies once you start changing the order of `#include`s (when sorting) - but of course this happens only once and does not prevent future or dependencies that left in the original order.

**Pull request merged**